packages/commercetools/theme/nuxt.config.js

```diff
@@ -20,15 +20,6 @@ export default {
         rel: 'icon',
         type: 'image/x-icon',
         href: '/favicon.ico'
-      },
-      {
-        rel: 'preconnect',
-        href: 'https://fonts.gstatic.com',
-        crossorigin: 'crossorigin'
-      },
-      {
-        rel: 'stylesheet',
```

Comment:
Why not add `preload` of stylesheets with fonts instead of adding new package?
Try with adding above that line:
```
{
        rel: 'preload',
        as: 'style',
        href: 'https://fonts.googleapis.com/css?family=Raleway:300,400,400i,500,600,700|Roboto:300,300i,400,400i,500,700&display=swap'
      },
```

Reply:
This is exactly what we removed. Lighthouse recommended that these requests for styles and fonts as a part of head section are render blocking. When using nuxt package we get rid of that render blocking fonts request. Also the package adds more improvements and optimalizations out of the box.

Reply:
![image](https://user-images.githubusercontent.com/37120330/133264486-71720347-693e-44ee-919d-2e5fb1a727c9.png)

Reply:
From what i see `preconnect` was removed for `https://fonts.gstatic.com` and stylesheets with `rel: 'stylesheet'`. Loading those styles without `preload` will be blocking but with `preload` it shouldn't be ;)